components/header/Header.tsx

```diff
@@ -1,62 +1,35 @@
 import { Box, Container, Flex } from "@chakra-ui/react";
-import { motion, useTransform } from "framer-motion";
 import Link from "next/link";
-import { useContext } from "react";
 
-import { colors } from "../../theme/colors";
 import Logo from "../Logo";
 import NavigationWrapper from "../navigation/NavigationWrapper";
-import { ScrollContext } from "./scrollContext/scrollContext";
 
-const setOpacity = (hex: string, alpha: number) =>
-  `${hex}${Math.floor(alpha * 255)
-    .toString(16)
-    .padStart(2, "0")}`;
-
-const Header = () => {
-  const { scrollYProgress } = useContext(ScrollContext);
-  const bgColor = useTransform(
-    scrollYProgress,
-    [0.7, 1],
-    [setOpacity(colors.accent, 0), colors.accent]
-  );
-
-  const textColor = useTransform(
-    scrollYProgress,
-    [0, 1],
-    [colors.primaryText, colors.black]
-  );
-
-  return (
-    <Box
-      as={motion.div}
-      position={"fixed"}
-      display={"flex"}
-      alignItems={"center"}
-      width={"full"}
-      style={{ backgroundColor: bgColor, color: textColor }}
-    >
-      <Container
-        backgroundColor={{ base: "accent", lg: "transparent" }}
-        maxWidth={"container.xl"}
+const Header = () => (
+  <Box display={"flex"} alignItems={"center"} width={"full"}>
+    <Container maxWidth={"container.xl"}>
```

Comment:
This could be flattened to Flex with wMax="container.xl"

Reply:
If possible i like to keep the container for readbility (it says in the name what it does)